libvast/src/command.cpp

```diff
@@ -69,7 +80,7 @@ caf::message run(const command& cmd, caf::actor_system& sys,
   bool has_subcommand;
   switch(state) {
     default:
-      return make_message(make_error(ec::unrecognized_option, *position));
+      return help_and_make_error_msg(cmd, ec::unrecognized_option, *position);
```

Comment:
Printing to STDERR is a side-effect the commands shouldn't have. Also, this hides the information we want to have later in shell output. Eventually, we also want to have a REST interface calling the commands. They would have no access the STDERR output but still would need the information which command got invoked. So probably the command name should get included into the message.

Reply:
I thought the exact same, but then realized that we're already printing to `std::cerr` in the commands anyway. Refactoring the side effect would thus be orthogonal to this PR.

Reply:
`run()` already has some writes to `std::cerr`. I consider this to be an orthogonal issue.

Reply:
It currently writes to `std::cerr` for historical reasons only and to me it makes no sense to pile onto technical debt. Any new code we write should look forward in terms of composability, rather than adding code we already know we need to revisit again.

Reply:
This story is not about refactoring the commands to be side-effect free. This is a bigger undertaking. Arguably, it's a dependency for this story but not in scope in my opinion.